### PR TITLE
[core] add fake secrets in tests to credscan

### PR DESCRIPTION
**Pull request by iscai-msft**

### CODE REVIEW

**Review by lmazuel — APPROVED** [review]

File: eng/CredScanSuppression.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
...
+                "SharedKey account:G4jjBXA7LI/RnWKIOQ8i9xH4p76pAQ+4Fs4R1VxasaE="
```
Comment (weshaggard) [inline eng/CredScanSuppression.json:35]: Is it possible to reuse an existing placeholder instead of adding a new one?

**Pull request merged**